csharp/src/Ice/IceDiscovery/Lookup.cs

```diff
@@ -41,7 +46,7 @@ public void FindAdapterById(string domainId, string adapterId, ILookupReplyPrx?
                 try
                 {
                     Debug.Assert(reply != null);
-                    reply.FoundAdapterByIdAsync(adapterId, proxy, isReplicaGroup);
+                    reply.FoundAdapterByIdAsync(adapterId, proxy, isReplicaGroup, cancel: cancel);
```

Comment:
You're using a named parameter here to skip the defaulted context parameter?

Reply:
yes